src/main/java/com/hbm/inventory/recipes/AssemblerRecipes.java

```diff
@@ -696,6 +696,12 @@ private static void registerDefaults() {
 				new ComparableStack(ModItems.motor, 2),
 			}, 200);
 
+		makeRecipe(new ComparableStack(ModItems.pipette_laboratory, 1),new AStack[] {
```

Comment:
Feels a bit tedious to make in an assembler, especially for what is essentially just a worse pipette. Maybe use the anvil instead? Also, what are the circuits for?

Reply:
The circuits are there because the laboratory pipette is considered a form of "micropipette", which have systems/mechanisms much more complex than normal pipettes.